jenkins/tippecanoe_create_tiles_Jenkinsfile

```diff
@@ -11,6 +11,7 @@ pipeline {
         sh 'wget -O DOIRootCA2.cer http://sslhelp.doi.net/docs/DOIRootCA2.cer'
         git "https://github.com/usgs-makerspace/wbeep-processing"
         sh 'aws s3 sync s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_shape . --exclude "*" --include "hrus.geojson"'
+        sh 'aws s3 sync s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_data . --exclude "*" --include "hru_data.csv"'
```

Comment:
Presumably there will be parallel job here that actually creates the data csv.  We'll need to be careful that we set up the triggers appropriately, and/or carefully scope what paths in the repo trigger rebuilds so that this doesn't run before the data csv is regenerated.

Reply:
yeah, i kind of just smooshed it in for now.. but imagine we might want to have it separate and triggered in a different way altogether once we understand how we'll get the data, what it looks like and what kind of pre-processing it needs?

Reply:
yeah true.  I'll go ahead and merge for now since this is temporary.

Reply:
🙏